source/sdk/lang/equalities.ooc

```diff
@@ -0,0 +1,47 @@
+// The following functions were moved from HashMap.ooc
```

Comment:
Yes, you are right. But then we don't want global functions. In this case let it slide as we don't want any of this code in the long run.